pfio/MultiGroupServer.F90

```diff
@@ -231,7 +247,7 @@ subroutine start_front()
          call this%report_profile()
```

Comment:
It looks like `report_profiler()` might have an RC code...but maybe it's not worth checking?

Reply:
If it's got one, it should be checked.   Just to save the sanity of the next guy. 

Or, if we can prove that there is no path that generates an error, the RC code should be eliminated from the routine.